examples/blk/board/qemu_virt_aarch64/blk.system

```diff
@@ -41,23 +41,24 @@
     <protection_domain name="virt" priority="99">
         <program_image path="blk_virt.elf" />
 
-        <map mr="blk_driver_config" vaddr="0x40000000" perms="rw" cached="false" setvar_vaddr="blk_config_driver"     />
+        <map mr="blk_driver_storage_info" vaddr="0x40000000" perms="rw" cached="false" setvar_vaddr="blk_storage_info_driver"     />
         <map mr="blk_driver_request" vaddr="0x40200000" perms="rw" cached="false" setvar_vaddr="blk_req_queue_driver"  />
         <map mr="blk_driver_response" vaddr="0x40400000" perms="rw" cached="false" setvar_vaddr="blk_resp_queue_driver" />
         <map mr="blk_driver_data" vaddr="0x40600000" perms="rw" cached="false" setvar_vaddr="blk_data_driver" />
-        <setvar symbol="blk_data_driver_paddr" region_paddr="blk_driver_data" />
+        <setvar symbol="blk_data_paddr_driver" region_paddr="blk_driver_data" />
```

Comment:
`blk_data_driver_paddr` makes more sense to me.

Reply:
To be consistent with everything else I guess it should be `blk_driver_data_paddr` and `blk_client_data_paddr`?

Reply:
Yeah this is a general issue with all of sddf where the ordering of the <system name>, <pd_name>, <purpose_of_variable> is done in whichever order as far as I can see. This applies similarly to the queue function names aswell e.g. **blk_queue_full_req**. I initially had in mind **blk_req_queue_full**, but was changed to make consistent with serial and network queues.

I reckon going from "big" to "small" is best here, like you've suggested blk_client_data_paddr makes sense. Just to be very clear I'm going for the approach of <system_name>, <pd_name>, <purpose> going forward. I'll make these changes only in the sdf file for now, in particular with the patched variable names. As for the rest of the system maybe this warrants a separate PR